src/integration-test/tests/citizen/endToEnd/steps/helper.ts

```diff
@@ -3,9 +3,11 @@ import { DefenceSteps } from 'integration-test/tests/citizen/defence/steps/defen
 import { DefenceType } from 'integration-test/data/defence-type'
 import { PaymentOption } from 'integration-test/data/payment-option'
 import { EndToEndTestData } from 'integration-test/tests/citizen/endToEnd/data/EndToEndTestData'
+import { DefendantSignSettlement } from '../../defence/pages/defendant-sign-settlement'
```

Comment:
please replace with correct path: example in the line above integration-test/tests/citizen